helm-chart/amalthea/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.1-set.by.chartpress
+version: 0.1.0
```

Comment:
This will be correctly set by chartpress before publishing the chart. However, I think increasing this number is a good idea. But then we'd have to do it by setting `resetVersion` in `chartpress.yaml` to something like `0.1.0-dev` now and bump it there with every release. Just bumping the number here will not be long lived as chartpress will overwrite it with `0.0.1-set.by.chartpress` in the one action.

Reply:
See https://github.com/jupyterhub/chartpress#configuration for reference